### DISCUSSION TITLE
item_category_name의 앞부분을 자르고 그 목록을 라벨인코딩 하면 어떨까 라고 생각해봤어요

### DISCUSSION THREAD

**Discussion by Seobbi** [Jung110/kaggleteam#15] [Ideas]
train에 존재하지 않는 test값을 예측하기 위해 item_category_id를 사용한다. 라고 양섭님이 정리하셨길래

item_category의 앞부분만 자르고 새롭게 만든 열을 라벨인코딩 (악세사리 >> 1) 이런식으로
item_id와 연결되어있는 item_category를 새롭게 만든 열로 연결하여 test값을 예측한다면? 라는 생각을 해봤어요

**Comment by Jung110** [comment]
넵 좋은 방법인 것 같습니다 다만 앞부분에 나눌 수 있는 기준이 존재하지 않는 경우도 생각해야 할 것 같습니다

**Comment by Jung110** [comment]
좀 더 다듬어서  한 것을 메인에 올려 놓았습니다.  
- 특수문자를 제거 후 split을 하였습니다. 
- 자동화 함수를 만들어서 shop_name에도 적용하면 좋을 것 같습니다.
